pages/pools/pools.tsx

```diff
@@ -0,0 +1,116 @@
+        </div>
+        <div className={style.topBarListings}>
+          <div className={style.topBarSingleListings}>
+            <p className={style.topBarData}>Total Pool Size</p>
+            <p className={style.topBarSingleValue}>
+              <span className={style.currencySymbol}>$</span>5.60B
+            </p>
+          </div>
+          <div className={style.topBarSingleListings}>
+            <p className={style.topBarData}>Total Available</p>
+            <p className={style.topBarSingleValue}>
+              <span className={style.currencySymbol}>$</span>5.60B
+            </p>
+          </div>
+          <div className={style.topBarSingleListings}>
+            <p className={style.topBarData}>Total Borrow</p>
+            <p className={style.topBarSingleValue}>
+              <span className={style.currencySymbol}>$</span>5.60B
+            </p>
+          </div>
+        </div>
+      </div>
+      <div className={style.tableView}>
+        <div className={style.topHeading}>
+          <div className={style.poolHeading}>
+            <p>Gammaswap assets</p>
+          </div>
+          <div className={style.createPool}>
+            <Link href={"/pools/createpool"}>
+              <div className={style.createPoolBtn}>Create Pool</div>
+            </Link>
+          </div>
+        </div>
+        <table className={style.tableData}>
+          <thead className={style.tableHead}>
+            <tr>
+              <th className={style.tableHeading}>Asset</th>
+              <th className={style.tableHeading}>Total Supply</th>
+              <th className={style.tableHeading}>Supply APY</th>
+              <th className={style.tableHeading}>Total Borrowed</th>
+              <th className={style.tableHeading}>
+                <div className={style.tableHeadDiv}>
+                  Borrow APY, variable <HiOutlineInformationCircle className={style.infoSvg} />
+                </div>
+              </th>
+              <th className={style.tableHeading}>
+                <div className={style.tableHeadDiv}>
+                  Borrow APY, stable <HiOutlineInformationCircle className={style.infoSvg} />
+                </div>
+              </th>
+              <th className={style.tableHeading}></th>
+            </tr>
+          </thead>
+          <tbody>
+            {poolData && poolData.length > 0
+              ? poolData.map((pool, index) => {
+                  return (
+                    <tr className={style.tableBodyRow} key={index}>
+                      <td className={style.tableBodyDefinition}>{pool.asset}</td>
+                      <td className={style.tableBodyDefinition}>{pool.totalSupply}M</td>
+                      <td className={style.tableBodyDefinition}>{pool.supplyApy}%</td>
+                      <td className={style.tableBodyDefinition}>{pool.totalBorrowed}M</td>
+                      <td className={style.tableBodyDefinition}>{pool.borrowApyVariable}%</td>
```

Comment:
Would be useful to have each pool Row being mapped out in its own component, makes for cleaner code. Something called `<PoolTableRow />`

Reply:
Done